artemis/src/main/java/tech/pegasys/artemis/cli/deposit/EncryptedKeystoreWriter.java

```diff
@@ -56,18 +56,13 @@ public void writeKeys(final BLSKeyPair validatorKey, final BLSKeyPair withdrawal
     final KeyStoreData withdrawalKeyStoreData =
         generateKeystoreData(withdrawalKey, withdrawalKeyPassword);
 
-    saveKeyStore(
-        keystoreDirectory.resolve("validator_" + validatorKey.getPublicKey().toString() + ".json"),
-        validatorKeyStoreData);
-    saveKeyStore(
-        keystoreDirectory.resolve(
-            "withdrawal_" + withdrawalKey.getPublicKey().toString() + ".json"),
-        withdrawalKeyStoreData);
+    saveKeyStore(keystoreDirectory.resolve("validator.json"), validatorKeyStoreData);
+    saveKeyStore(keystoreDirectory.resolve("withdrawal.json"), withdrawalKeyStoreData);
   }
 
   private Path createKeystoreDirectory(final BLSKeyPair validatorKey) {
     final Path keystoreDirectory =
-        outputPath.resolve("validator_" + validatorKey.getPublicKey().toString());
+        outputPath.resolve("validator_" + validatorKey.getPublicKey().toString().substring(2, 9));
```

Comment:
instead of directly using substring, may I suggest to introduce a method something on the lines of :

~~~
if (publicKey.toLowerCase().startsWith("0x")) {
  return publicKey.substring(2,9); 
} else {
  return publicKey.substring(0, 7);
}
~~~

as the public key technically can start without "0x"